assets/dev/scss/frontend/general/_button.scss

```diff
@@ -25,16 +25,25 @@
 	}
 
 	&-icon {
+
+		i {
+			color: var(--button-icon-color, inherit);
+		    font-size: var(--button-icon-size);
+		}
+		
 		svg {
-			width: 1em;
+			width: var(--button-icon-size, 1em);
 			height: auto;
+			fill: var(--button-icon-color, inherit);
 		}
-
+		
```

Comment:
Remove tabs.

Reply:
Our code needs to be completely clean before it can be merged.

Reply:
@Mchris-hub Not fixed yet.

Reply:
@Mchris-hub One empty line has been removed, but new tabs have been introduced. This needs to be fixed.